### PR TITLE
Remove generic upgrades

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

**Comment by evilnick** [comment]
Fixes #707

File: pages/k8s/upgrading.md [hunk 1]
```diff
@@ -74,319 +74,24 @@ This may not include any customised configuration of Kubernetes, or user
...
-
-All of the charms support [upgrading the machine's series via Juju](https://juju.is/docs/upgrading-series).
-As each machine is upgraded, the applications on that machine will be stopped and the unit will
-go into a `blocked` status until the upgrade is complete. For the worker units, pods will be drained
-from the node and onto one of the other nodes at the start of the upgrade, and the node will be removed
-from the pool until the upgrade is complete.
+- [Upgrading from 1.20.x to 1.21.x](/kubernetes/docs/1.21/upgrading)
+- [Upgrading from 1.19.x to 1.20.x](/kubernetes/docs/1.19/upgrading)
+- [Upgrading from 1.18.x to 1.19.x](/kubernetes/docs/1.18/upgrading)
+- [Upgrading from 1.17.x to 1.28.x](/kubernetes/docs/1.17/upgrading)
```
Comment (Cynerva) [inline pages/k8s/upgrading.md:94]: I think there are a few typos in here, see suggestion:

```suggestion
- [Upgrading from 1.20.x to 1.21.x](/kubernetes/docs/1.21/upgrading)
- [Upgrading from 1.19.x to 1.20.x](/kubernetes/docs/1.20/upgrading)
- [Upgrading from 1.18.x to 1.19.x](/kubernetes/docs/1.19/upgrading)
- [Upgrading from 1.17.x to 1.18.x](/kubernetes/docs/1.18/upgrading)
```

File: pages/k8s/upgrading.md [hunk 2]
```diff
@@ -74,319 +74,24 @@ This may not include any customised configuration of Kubernetes, or user
 generated objects (e.g. storage classes) or deployments which rely on
 deprecated APIs.
 
-## Infrastructure updates
+## Specific upgrade instructions
 
-The applications which run alongside the core Kubernetes components can be upgraded at any time. These applications are widely used and may frequently receive upgrades outside of the cycle of new releases of Kubernetes.
+We have never recommended skipping a version in an upgrade and no longer provide generic upgrade 
+instructions for any version of Charmed Kubernetes. Please follow the specific upgrade 
+documentation for the version you are upgrading to
```
Comment (Cynerva) [inline pages/k8s/upgrading.md:81]:
```suggestion
documentation for the version you are upgrading to.
```

**Review by Cynerva — CHANGES_REQUESTED** [review]
I have a couple nitpicky suggestions, but this looks great overall and I think it should make things a lot clearer. Thanks :+1:

File: pages/k8s/upgrading.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/upgrading.md:94]: Gosh, I made a mess of that!

**Review by Cynerva — APPROVED** [review]

**Pull request merged**